src/context/components/metatags.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Helmet} from 'react-helmet';
+
+const MetaTags = ( {post} ) => {
+    // Note: The manifest is the 'creat-react-app' manifest.
+    // Adding it in the head wouldn't matter until it were filled out.
+    // basically, if someone wants a shortcut to your blog on their phone screen,
+    // the manifest will provide information about the image, name, etc for the resulting icon, link, and blah.
+   return(   
+        post ? (
+        <Helmet>
```

Comment:
if the MetaTags component is passed a post prop,
define a bunch of tags for the head. In this case, I was almost exclusively concerned with "social shares".
(So, getting an image, proper title, description, etc., when sharing a post on social media).